src/storage/src/hummock/iterator/mod.rs

```diff
@@ -160,6 +165,10 @@ impl<D: HummockIteratorDirection> HummockIterator for PhantomHummockIterator<D>
     }
 
     fn collect_local_statistic(&self, _stats: &mut StoreLocalStatistic) {}
+
+    fn value_meta(&self) -> Option<u64> {
```

Comment:
nites: given that we call it `value_meta`, I think we can return a `ValueMeta` struct instead:
```
struct ValueMeta {
  object_id: i64
}
```

We may want to include more things here in the future.